example/module/sequential_module.py

```diff
@@ -0,0 +1,60 @@
+# pylint: skip-file
+import mxnet as mx
+import numpy as np
+import logging
+
+# whether to demo model-parallelism + data parallelism
+demo_data_model_parallelism = True
+
+if demo_data_model_parallelism:
+    contexts = [[mx.context.gpu(0), mx.context.gpu(1)], [mx.context.gpu(2), mx.context.gpu(3)]]
+else:
+    contexts = [mx.context.cpu(), mx.context.cpu()]
+
```

Comment:
@mli @piiswrong here is a very simple hybrid model-par + data-par example. The whole network is split into two parts, the first part is computed in gpu0, gpu1, and the second part computed in gpu2, gpu3. We already have model-par in symbolic computation graph as in @tqchen 's lstm example. But I guess doing hybrid is a bit complicated in that API.